src/App.tsx

```diff
@@ -1,16 +1,25 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
+
+import { Movie } from './types/Movie';
+
 import moviesFromServer from './api/movies.json';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+  const handleAddMovie = (movie: Movie) => {
+    setMovies([...movies, movie]);
```

Comment:
Better to use "previous" value of state in such cases:
```suggestion
    setMovies(prevMovies => [...prevMovies, movie]);
```